build/project_cleanup.yaml

```diff
@@ -0,0 +1,29 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+timeout: 3600s
+
+options:
+  machineType: "N1_HIGHCPU_8"
+
+steps:
+- name: 'gcr.io/cloud-builders/gcloud'
+  entrypoint: 'bash'
+  args: ["scripts/project_cleanup.sh"]
+  id: Project deletion
+  env:
+  - "FOLDER_ID=${_FOLDER_ID}"
+options:
+  logging: CLOUD_LOGGING_ONLY
+  
```

Comment:
Add a new line at the end

Reply:
Do I need at least two newlines at the end?

Reply:
hmm one line should be fine. The latest commit you pushed looks good.